index.html

```diff
@@ -5,11 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Telemetry portal</title>
-
+   
```

Comment:
Looks as though there might be some unrelated whitespace changes to this file? (git tip: try `git checkout origin/master -- path/to/file` to change a file back to an unedited version

Reply:
Okay